pkg/virt-handler/node-labeller/model.go

```diff
@@ -95,3 +95,8 @@ type SEVConfiguration struct {
 	Cbitpos         string `xml:"cbitpos"`
 	ReducedPhysBits string `xml:"reducedPhysBits"`
 }
+
+type KSMConfiguration struct {
+	SysfsFilePath string
+	Enabled       bool
+}
```

Comment:
Not sure that this struct is needed here SysfsFilePath is const (ksmPath) if i understood it correctly
In my opinion a bool would be enough

Reply:
I introduced this struct for two main reasons:
1. Allow to easily introduce future enhancements like adding custom labels based on other ksm sysfs interface (https://www.kernel.org/doc/html/latest/admin-guide/mm/ksm.html#ksm-daemon-sysfs-interface)
2. facilitate unit testing (since `SysfsFilePath` is tunable we can create a mock nodeLabeller with a temporary file in which we can write whatever we want, without overriding local/host environment)